modules/ROOT/pages/partner-manager-identifiers.adoc

```diff
@@ -87,17 +87,17 @@ Use the following values to configure the X12 GS identifier type:
 [[reference-id]]
 == Outbound Message Flow Partner Reference Identifiers
 
-Outbound message flows that transform your backend application messages to EDI format use the partner Reference ID when sending data to your partners.
+Outbound message flows that transform your backend-application messages to EDI format use the partner Reference ID when sending data to your partners.
 
-Unlike EDI transactions that have well-defined standards for how sender, receiver, and document type information is presented in the data, the source messages for outbound B2B data exchange originating from your enterprise backend systems typically follow schemas. These schemas are usually defined by your backend application teams according to your organization’s data models.
+Unlike EDI transactions that have well-defined standards for how sender, receiver, and document type information is presented in the data, the source messages for outbound B2B data exchange originating from your enterprise backend systems typically follow schemas. These schemas are usually defined by your back-end application teams according to your organization’s data models.
```

Comment:
```suggestion
Unlike EDI transactions that have well-defined standards for how sender, receiver, and document type information is presented in the data, the source messages for outbound B2B data exchange originating from your enterprise back-end systems typically follow schemas. These schemas are usually defined by your back-end application teams according to your organization’s data models.
```

Reply:
Disregard if I'm wrong. I always get confused about when it's backend vs back-end.

Reply:
You are right.